docs/content/concepts/object-ownership/party.mdx

```diff
@@ -1,39 +1,39 @@
 ---
 title: Party Objects
-description: Party objects are owned by a specified party given at the time of transfer, and versioned by consensus. Learn how to create and access these objects.
+description: Party objects are owned by a specified party at the time of transfer and versioned by consensus. Learn how to create and access these objects.
 ---
 
 A party object is an object that is transferred using the `sui::transfer::party_transfer` or `sui::transfer::public_party_transfer` function. It is accessible to the `Party` to which it is transferred.
 
-Why the name "party object"? Think "party" as in a dining party at a restaurant, or a party of intrepid adventurers.
+The name party object refers to the idea of a dining party at a restaurant or a party of adventurers.  
```

Comment:
Thank goodness you came across this sentence. My og suggestion was ignored. I think it can be removed as it's really just restating the sentence before.